Stripe.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
once last thing, this shouldn't be committed (it also shouldn't even be tracked by git, we'll look into this).